<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>safeQuestion</class>
 <widget class="QDialog" name="safeQuestion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>safeQuestion</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic; color:#ff5500;&quot;&gt;安 全 问 题 验 证&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="typeCombox">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>69</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>教师</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>学生</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="typeLab">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>120</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>用户类型：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okBtn">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelBtn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QLabel" name="questionLab">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>安全问题：</string>
   </property>
  </widget>
  <widget class="QLabel" name="answerLab">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>250</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>答   案：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="answerlineEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>250</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="questionlineEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>200</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="idlineEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="idLab">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>用户编号：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="checkBtn">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>150</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>验证</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>cancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>safeQuestion</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>328</x>
     <y>296</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
